cc-env.go

```diff
@@ -97,9 +97,8 @@ type ShimInfo struct {
 
 // AgentInfo stores agent details
 type AgentInfo struct {
-	Type         string
-	Version      string
-	PauseBinPath string
+	Type    string
```

Comment:
As you are changing the output of `cc-env`, please can you bump `formatVersion` in `cc-env.go`.

Reply:
Done